spec.html

```diff
@@ -14806,7 +14806,7 @@ <h1>Module Namespace Exotic Objects</h1>
               a List of Strings
             </td>
             <td>
-              A List whose elements are the String values of the exported names exposed as own properties of this object. The list is ordered as if an Array of those String values had been sorted using %Array.prototype.sort% using *undefined* as _comparefn_.
+              A List whose elements are the String values of the exported names exposed as own properties of this object. The List is ordered as if an Array of those String values had been sorted using %Array.prototype.sort% using *undefined* as _comparefn_.
```

Comment:
I would not capitalize "The List" here either.

Reply:
Thanks for the review! All code review problem is done.